GermanFilter/sections/general_extensions.txt

```diff
@@ -21,6 +21,9 @@ chip.de#%#//scriptlet('set-cookie', 'chipde-r2d2-view-state ', '1')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/135749
 ! https://github.com/AdguardTeam/AdguardFilters/issues/135687
 giga.de,spiegel.de,welt.de,t-online.de#%#(()=>{window.XMLHttpRequest.prototype.send=new Proxy(window.XMLHttpRequest.prototype.send,{apply:(a,b,c)=>{const d=b.urlCalled;return"string"!=typeof d||0===d.length?Reflect.apply(a,b,c):d.match(/\.damoh\./)?void 0:Reflect.apply(a,b,c)}})})();
+!#if (adguard_ext_firefox)

```

Comment:
Using hint `!+ PLATFORM(ext_ff,)` is enough here.
Seems the rule works fine, I haven't seen the incorrect blocking in the videos.